benchmarks/uhs_leveldb.cpp

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2021 MIT Digital Currency Initiative,
+//                    Federal Reserve Bank of Boston
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "uhs/transaction/messages.hpp"
+#include "uhs/transaction/transaction.hpp"
+#include "uhs/transaction/validation.hpp"
+#include "uhs/transaction/wallet.hpp"
+
+#include "util/serialization/istream_serializer.hpp"
+#include "util/serialization/ostream_serializer.hpp"
+
+#include <benchmark/benchmark.h>
+#include <filesystem>
+#include <gtest/gtest.h>
+#include <leveldb/db.h>
+#include <leveldb/write_batch.h>
+
+#include <vector>
```

Comment:
I know you ran into some issues with the Fixture and threading for levelDB; is there another way of grouping benchmarks in leveldb? It would be nice if these could be grouped similarly to how the Fixture groups them for low-level.

Reply:
Good question, I will need to look into this more, but one idea I'll start with is uniting them with a class -- in the output I would assume they will be listed in class/subclass format just as the Fixtures do